### PR TITLE
Immolate change and particles

### PR DESCRIPTION
**Pull request by ezrn**
- Made it identical to mineplex(gives strength but less speed)
- Also added some cool fire particles

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java [hunk 1]
```diff
@@ -35,15 +36,12 @@
...
-
     private double fireTickDurationIncreasePerLevel;
-
     private double baseFireTrailDuration;
-
     private double fireTrailDurationIncreasePerLevel;
-
     private int speedStrength;
-
+    private int strengthStrength;
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java:43]: im sure you can think of a better name

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java [hunk 2]
```diff
@@ -199,10 +205,10 @@ private void sendState(Player player, boolean state) {
     public void loadSkillConfig() {
         baseFireTickDuration = getConfig("baseFireTickDuration", 4.0, Double.class);
         fireTickDurationIncreasePerLevel = getConfig("fireTickDurationIncreasePerLevel", 0.0, Double.class);
-
         baseFireTrailDuration = getConfig("baseFireTrailDuration", 2.0, Double.class);
         fireTrailDurationIncreasePerLevel = getConfig("fireTrailDurationIncreasePerLevel", 0.0, Double.class);
-
-        speedStrength = getConfig("speedStrength", 1, Integer.class);
+        speedStrength = getConfig("speedStrength", 0, Integer.class);
+        strengthStrength = getConfig("speedStrength", 1, Integer.class);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java:211]: 🤔

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java [hunk 3]
```diff
@@ -199,10 +205,10 @@ private void sendState(Player player, boolean state) {
     public void loadSkillConfig() {
         baseFireTickDuration = getConfig("baseFireTickDuration", 4.0, Double.class);
         fireTickDurationIncreasePerLevel = getConfig("fireTickDurationIncreasePerLevel", 0.0, Double.class);
-
         baseFireTrailDuration = getConfig("baseFireTrailDuration", 2.0, Double.class);
         fireTrailDurationIncreasePerLevel = getConfig("fireTrailDurationIncreasePerLevel", 0.0, Double.class);
-
-        speedStrength = getConfig("speedStrength", 1, Integer.class);
+        speedStrength = getConfig("speedStrength", 0, Integer.class);
+        strengthLevel = getConfig("speedLevel", 1, Integer.class);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java:211]: ...

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java (hunk 3 above)
Comment (StudentAlleg) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java:211]: It technically works; better than some of my issues

Comment (braulio-dev) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java:211]: NICE ONE STUDENBT!

**Pull request merged**